package.json

```diff
@@ -6,7 +6,7 @@
   "module": "dist/index.es6.js",
   "types": "dist/index.d.ts",
   "scripts": {
-    "test": "jest --env=jsdom",
+    "test": "jest",
```

Comment:
Moved `--env=jsdom` to [`jest.testEnvironment`](https://facebook.github.io/jest/docs/en/configuration.html#testenvironment-string) configuration below to keep this with all other Jest configuration.